build.gradle

```diff
@@ -162,12 +162,22 @@ task generateApiRequestFormat(type: cz.habarta.typescript.generator.gradle.Gener
     outputFile = "src/web/types/api-request.ts"
 }
 
+task generateIntent(type: cz.habarta.typescript.generator.gradle.GenerateTask) {
+    classes = [
+        "teammates.ui.webapi.action.Intent"
+    ]
+    mapEnum = "asEnum"
+    outputFile = "src/web/types/Intent.ts"
```

Comment:
Instead, you should delete the `Intent.ts` and use `Intent` from `api-request.ts`

Reply:
@xpdavid  Deleted `Intent.ts`